### PR TITLE
Add internationalization with i18next

### PR DESCRIPTION
**Pull request by sergivv**
## Description
Add internationalization with i18next. 
Translations are added for 5 languages ​​(English, Spanish, Galician, Basque and Catalan)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/codevsoft/workkit/blob/master/CONTRIBUTING.md) doc document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: libs/page-login/src/lib/page-login.tsx [hunk 1]
```diff
@@ -61,14 +62,34 @@ const useStyles = makeStyles((theme) => ({
 	},
 }));
 
+function SelectLanguage() {
+	const { t, i18n } = useTranslation();
```
Comment (namelivia) [inline libs/page-login/src/lib/page-login.tsx:66]:
```suggestion
	const { i18n } = useTranslation();
```

Comment (sergivv) [reply libs/page-login/src/lib/page-login.tsx:66]: You're right, I don't know why the VSCODE didn't detect that unused variable.

File: apps/workkit-fe/src/main.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { I18nextProvider } from 'react-i18next';
+import i18n from './i18n';
 
 import App from './app/app';
 
 ReactDOM.render(
-  <React.StrictMode>
```
Comment (73nko) [inline apps/workkit-fe/src/main.tsx:7]: Why did you removed strict mode? Is It needed for I18 towork?

File: libs/page-login/src/lib/page-login.tsx [hunk 3]
```diff
@@ -61,14 +62,34 @@ const useStyles = makeStyles((theme) => ({
 	},
 }));
 
+function SelectLanguage() {
```
Comment (73nko) [inline libs/page-login/src/lib/page-login.tsx:65]: I think this should be a mes component which se dan reuse in other places.

File: libs/page-login/src/lib/page-login.tsx [hunk 4]
```diff
@@ -80,7 +101,7 @@ export const PageLogin = (): React.ReactElement => {
 						<LockOutlinedIcon />
 					</Avatar>
 					<Typography component="h1" variant="h5">
-						Acceso a la plataforma
+						{t('title')}
```
Comment (73nko) [inline libs/page-login/src/lib/page-login.tsx:104]: Could you add some unit tests to verify this is working properly.

**Comment by 73nko** [comment]
Great work, I've just added nit comments 👍

**Review by 73nko — APPROVED** [review]
Awesome work! 🎉 
LGTM! ✅

**Pull request merged**